server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +213,83 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$virtualAppointmentConsentId = insertConsent($reviewConsent, $virtualConsent, $signature);
+		$DB_CONN->commit();
+
+		$response['success'] = true;
+		$response['virtualAppointmentConsentId'] = $virtualAppointmentConsentId;
+		// $response['message'] = AppointmentConfirmationUtilities::generateAppointmentConfirmation($appointmentId);
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		$response['message'] = 'An error occurred while trying to record your consent. Please try again in a few minutes.';
+	}
+
+	print json_encode($response);
+}
+
+function addConsentForeignKeyToAppointment($appointmentId, $virtualAppointmentConsentId) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+
+	try {
+		$appointmentAccessor = new AppointmentAccessor();
+		$appointmentAccessor->addVirtualAppointmentConsentId($appointmentId, $virtualAppointmentConsentId);
+
+		$response['success'] = true;
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		$response['message'] = 'An error occurred while trying to record your consent. Please try again in a few minutes.';
+	}
+
+	print json_encode($response);
+}
+
 /* 
  * Private functions
  */
 
+function insertConsent($reviewConsent, $virtualConsent, $signature) {
+	GLOBAL $DB_CONN;
+
+	$consentInsert = 'INSERT INTO VirtualAppointmentConsent (reviewConsent, virtualConsent, signature)
+		VALUES (?, ?, ?)';
+	$consentParams = array($reviewConsent, $virtualConsent, $signature);
+
+	$stmt = $DB_CONN->prepare($consentInsert);
+	if(!$stmt->execute($consentParams)){
+		throw new Exception("There was an issue on the server. Please refresh the page and try again.", MY_EXCEPTION);
+	}
+
+	return $DB_CONN->lastInsertId();
+}
+
+
+function insertConsentForeignKeyToAppointment($appointmentId, $virtualAppointmentConsentId) {
+	GLOBAL $DB_CONN;
+
+	$consentInsert = 'INSERT INTO Appointment (virtualAppointmentConsentId)
```

Comment:
You'll remove this when you switch the FK direction--but this should've been an UPDATE statement, not an INSERT

Reply:
Thanks for lmk!!